bot/bot.go

```diff
@@ -111,8 +111,11 @@ func (b *Bot) Init(rtm *slack.RTM) error {
 
 	params := slack.PostMessageParameters{AsUser: true}
 	_, _, err = b.slackBotAPI.PostMessage(b.users["dlsniper"], fmt.Sprintf(`Deployed version: %s`, b.version), params)
+	if err != nil {
+		b.logf("Failed to send deploy version message.")
```

Comment:
This could be changed to:
```
b.logf(`failed to deploy version: %s`, b.version)
```

As that would help out to identify which version failed to be deployed.